### PR TITLE
Enable passing args into function.

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: lib/LruMemoize.js [hunk 1]
```diff
@@ -27,17 +27,19 @@ export default class LruMemoize {
    * @param {string} options.key - A key for the cache.
    * @param {Function<Promise>} options.fn - A Function that returns a
    *   promise to memoize.
+   * @param {object} [options.args] - The arguments to pass into the function
+   *   `fn`.
    *
    * @returns {Promise} - The result of the memoized promise.
   */
-  async memoize({key, fn}) {
+  async memoize({key, fn, args}) {
```
Comment (mattcollier) [inline lib/LruMemoize.js:35]: the current design is that `fn` would be a closure which is how its currently being used... for example
```
const arg1 = 'foo';

const fn = () => someFunction({arg1});

await lruMemoize.memoize({key: 'myKey', fn})
```

Comment (aljones15) [reply lib/LruMemoize.js:35]: we should probably provide that as `@example` so future devs see it.

**Comment by dlongley** [comment]
-1 to this approach -- it's making the API more complicated due to an optimization strategy that may not matter in the near future.

**Comment by gannan08** [comment]
Closing. Closure pattern is the preferred way to use the library.

**Pull request closed without merging**